.github/workflows/committed-chart.yaml

```diff
@@ -0,0 +1,33 @@
+name: Chart Generation
+
+
+on:
+  pull_request:
```

Comment:
Would it make sense to add a `paths` filter here to run only when relevant files change? Like `charts/**`

Reply:
This makes sure that charts/** has been built, so if you wanted to limit its scope you could do it to .helm/, however there may be other edge cases that I am not thinking of which would require the chart to be regenerated